src/templates/player.js

```diff
@@ -1,6 +1,6 @@
-export default (id) => {
+export default (id, ariaLabel) => {
```

Comment:
Add a default argument of `''` in case `ariaLabel` is undefined (`aria-label="undefined"` looks bad)